README.md

```diff
@@ -30,33 +30,29 @@ Clone the repository.
 
 | Directory  | Chapter |
 | ------------- | ------------- |
-| Chapter 1 - Introduction | src/main/scala/com/professionalscala/ch1 |
-| Chapter 2 - Functional Programming | src/main/scala/com/professionalscala/ch2 |
-| Chapter 3 - Concurrency | src/main/scala/com/professionalscala/ch3 |
-| Chapter 3 - Scala and Java Collections | src/main/scala/com/wrox/professionalscala/ch03 |
+| Chapter 1 - Introduction | src/main/scala/com/wrox/professionalscala/ch01 |
+| Chapter 2 - Functional Programming | src/main/scala/com/wrox/professionalscala/ch02 |
+| Chapter 3 - Java Compatibility | src/main/scala/com/wrox/professionalscala/ch03 |
 | Chapter 4 - SBT | chapter4_codesamples |
 | Chapter 5 - Maven | chapter-5 |
 | Chapter 7 - Testing | chapter7_codesamples |
 | Chapter 8 - Scaladoc | chapter-8 |
 | Chapter 9 - Type System | src/main/scala/com/wrox/professionalscala/ch09 |
 | Chapter 10 - Advanced Functional Programming | src/main/scala/com/wrox/professionalscala/ch10 |
-| Chapter 11 - Java Compatibility | src/main/scala/com/professionalscala/ch11 |
+| Chapter 11 - Java Compatibility | src/main/scala/com/wrox/professionalscala/ch11 |
 
 ## Usage
 
-### Chapters 1, 2, 4, 7, 11 and some of Chapter 3
+### Chapters 1, 2, 3, 4, 7, 8, 9, 10 and 11
 
-Run the examples using `sbt` on the  command line (sbt run  will do the trick).
-
-### Chapters 9, 10 and some of Chapter 3
-
-Launch sbt. Then from the sbt prompt launch the `runMain` command followed by the example you want to run. E.g.:
+Run the examples using `sbt` on the  command line (sbt run  will do the trick). Alternatively, from the sbt prompt,
```

Comment:
Looks good to me, although isn't it better to just use `run` instead of `runMain` (since the SBT repl will allow you to select which main class to run, i.e. if I do `run` with my project I get)

```
Multiple main classes detected, select one to run:

 [1] com.professionalscala.ch1.AbstractClass
 [2] com.professionalscala.ch1.CaseClasses
 [3] com.professionalscala.ch1.Collections
 [4] com.professionalscala.ch1.ForComprehension
 [5] com.professionalscala.ch1.ImplicitClasses
 [6] com.professionalscala.ch1.ImplicitParameters
 [7] com.professionalscala.ch1.IntsVsDoubles
 [8] com.professionalscala.ch1.IntsVsDoublesProper
 [9] com.professionalscala.ch1.LocalTypeInference
 [10] com.professionalscala.ch1.ManualTypeInference
 [11] com.professionalscala.ch1.ManualTypeSignatures
 [12] com.professionalscala.ch1.ManualTypeSignatures2
 [13] com.professionalscala.ch1.PackagesCompanionObjectAndScoping
 [14] com.professionalscala.ch1.PatternMatching
 [15] com.professionalscala.ch1.Trait
 [16] com.professionalscala.ch1.TypeAnnotations
```